src/NServiceBus.Metrics/Core/ProbeBuilder.cs

```diff
@@ -0,0 +1,15 @@
+namespace NServiceBus.Metrics
+{
+    using Features;
+
+    /// <summary>
+    /// Defines a custom metric.
+    /// </summary>
+    abstract class ProbeBuilder
```

Comment:
why an abstract class instead of an interface? given it has no implementation methods